react-validated-input-component/dist/styles.css

```diff
@@ -0,0 +1,21 @@
+* {
+  box-sizing: border-box;
+}
+
+form {
+  margin-right: 10px;
+}
+
+.fa-check {
+  color: green;
+  margin-left: 15px;
+}
+
+.fa-times {
+  color: red;
+  margin-left: 15px;
+}
```

Comment:
Avoid styling 3rd-party CSS classes. This changes their behavior and the docs can't be relied upon by your team members anymore.

Instead, create additional classes of your own to use along with these.